### PR TITLE
Routing update and overall improvements

### PR DESCRIPTION
**Pull request by kgish**
While reading the section on routing as a (native English), I made a number of corrections and fixed some typos which I believe will improve the overall readability of the text. Hopefully I didn't mess up any of the technical details, but ran the code to double-check so think it's alright.

### CODE REVIEW

**Comment by lancehalvorsen** [comment]
@kgish Thanks so much for your work on this! 

Github is reporting merge conflicts from your changes (without showing me where). Could you pull in upstream master and see if that fixes them? Thanks in advance.

**Comment by kgish** [comment]
I'm a bit of a newbie when it comes to doing fancy git stuff like this. So feeling brave to try the very best of my abilities, I did the following:
```bash
$ git pull https://github.com/lancehalvorsen/phoenix-guides.git
<fixed the merge conflicts>
$ git add .
$ git commit
$ git push origin master
```
But not sure if correct, is it?

**Comment by lancehalvorsen** [comment]
@kgish unfortunately, there are still merge conflicts and github will not let me merge.

Question: did you originally fork my repo or clone it?

**Comment by kgish** [comment]
Let me try again with another pull request and see if that works.
